content/en/truecommand/overview/tcwhatis.md

```diff
@@ -4,9 +4,51 @@ description: "Describing the features and some use cases of the TrueCommand NAS
 weight: 10
 ---
 
-TrueCommand is a single Pane-of-Glass management application that takes the repetitive work out of multi-TrueNAS management by centralizing system alerts, reports, and analytics in one easy to use interface.
-It supports users and teams with 24x365 global operations, role-based access control, and a full log of every TrueNAS configuration change.
-TrueCommand runs on docker, VMs, or as a cloud service and is free to users with less than 50 drives and affordable for those with larger installations.
+### What is TrueCommand?
 
-<img src="/images/truecommand-overview.png">
+TrueCommand is a multi-system management "Single pane of Glass" system that helps control and monitor your TrueNAS fleet. TrueCommand assists in managing TrueNAS systems through REST APIs, WebSocket APIs, and a web user interface. The TrueCommand web interface provides unified administration of users and TrueNAS systems and single sign-on functionality.
```

Comment:
should "provides unified" be   "provides a unified"?

Reply:
yup, i missed the 'a'

Reply:
This is another, #english_grammar_is_weird moment.  I've changed the sentence around a bit.